### ISSUE TITLE
\textbf instead of \section

### ISSUE THREAD

**Issue by garyritchie** [transpect/docx2tex#106]
Until now I've just been using docx2tex to set some MathType free (_thank you!_) but now I;ve got another project and I've noticed that headings aren't translating to sections. I was expecting `\chapter`, `\section` and `\subsection` but I'm getting `\textbf` instead. Here's a minimal test output:

`d2t test_d2t.docx` gives:

```latex
% docx2tex 1.9 --- ``How I learned to stop worrying and love DOCX'' 
% 
% docx2tex is Open Source and  
% you can download it on GitHub: 
% https://github.com/transpect/docx2tex 
%  
\documentclass{scrbook} 
\usepackage[T1]{fontenc} 
\usepackage[utf8]{inputenc} 
\usepackage{hyperref} 
\usepackage{multirow} 
\usepackage{tabularx} 
\usepackage{color} 
\usepackage{textcomp} 
\usepackage{tipa}
\usepackage{amsmath} 
\usepackage{amssymb} 
\usepackage{amsfonts} 
\usepackage{amsxtra} 
\usepackage{wasysym} 
\usepackage{isomath} 
\usepackage{mathtools} 
\usepackage{txfonts} 
\usepackage{upgreek} 
\usepackage{enumerate} 
\usepackage{tensor} 
\usepackage{pifont} 
\usepackage{ulem} 
\usepackage{xfrac} 
\usepackage{arydshln} 
\usepackage[english]{babel}

\begin{document}
\textbf{Test Document Heading 1}

Some paragraph text

\textbf{A Heading Level 2}

More paragraph text
\end{document}
```

The minimal LibreOffice-created docx is attached. 

[test_d2t.docx](https://github.com/user-attachments/files/16133849/test_d2t.docx)

There are two errors in the log (attached below):

```log
...
[main] ERROR com.xmlcalabash.runtime.XAtomicStep - file:/docx2tex/xproc-util/load/xpl/load.xpl:0:load-error:Could not load file:///docx2tex/conf/conf.csv (file:/docx2tex/xproc-util/load/xpl/load.xpl) dtd-validate=false
[main] ERROR com.xmlcalabash.runtime.XAtomicStep - file:/docx2tex/xproc-util/load/xpl/load.xpl:0:load-error:Could not load file:///docx2tex/conf/conf.csv (file:/docx2tex/xproc-util/load/xpl/load.xpl) dtd-validate=false
...
```
I did edit conf.csv with VSCodium to match the style names:
```csv
Heading 1	; \chapter{	; }
Quotations 	; \begin{quote}	; end{quote}
Heading 2	; \section{	; }
Heading 3	; \subsection{	; }
```

[test_d2t.d2t.log](https://github.com/user-attachments/files/16133875/test_d2t.d2t.log) (paths shortened)

EDIT: Added error log; command used.

**Comment by mkraetke** [comment]
Thank you for the report! There were two problems: 
1. The README.md did not state that the style names must apply to the `role` attributes in the generated XML file. I specified this more precisely.
2. The XSLT that converts the CSV internally to the XML configuration format replaced underscores and did not normalized it before. This issue is now resolved, too.

If you pull my latest fixes and change the configuration as specified below, it should work:

```csv
Heading_1 ; \chapter{ ; }
Quotations ; \begin{quote} ; end{quote}
Heading_2 ; \section{ ; }
Heading_3 ; \subsection{  ; }
```
